.github/workflows/ci.yml

```diff
@@ -83,6 +83,26 @@ jobs:
       - run: cargo clippy --all-targets -- -D warnings
         working-directory: rust
 
+  cargo-test:
+    needs: changes
+    if: ${{ needs.changes.outputs.rust == 'true' }}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Setup rust toolchain
+        run: rustup show
+      - uses: Swatinem/rust-cache@v2.1.0
+      - uses: subosito/flutter-action@v2
+        with:
+          channel: "stable"
+      - name: Install FFI bindings
+        run: cargo install flutter_rust_bridge_codegen
```

Comment:
oh boy, install ffi bindings again 🙈 we need to tackle https://github.com/itchysats/10101/issues/79